package.json

```diff
@@ -33,7 +33,8 @@
     "test:embark": "mocha test/ --no-timeouts --exit",
     "test:test_app": "cross-env DAPP=\"test_app\" npm run test_dapp",
     "test:contracts_app": "cross-env DAPP=\"contracts_app\" npm run test_dapp",
-    "test_dapp": "cross-env-shell \"cd test_apps/$DAPP && npm install && npm test\""
+    "test_dapp": "cross-env-shell \"cd test_apps/$DAPP && npm install && npm test\"",
+    "release": "./scripts/release"
```

Comment:
With our growing collection of npm-scripts, I think it's helpful to maintain alpha order so it's easier to visually find a particular script.

Reply:
Good point. I'll update this.